### ISSUE TITLE
Will test data contain .gitignore ?

### ISSUE THREAD

**Issue by MarkYan** [SytemProgramming2015FallatNTU/Announcement#49]
I found that .gitignore seem to be a tricky file, 
it will update as you modify other file.

Will the Test data contain .gitignore?

**Comment by kucw** [comment]
No, we'll replace your cdir before the test, so you don't need to handle .gitignore.
